### PR TITLE
util: migrate importer and watcher from tidb-tools to tidb

### LINKED ISSUE #32999: migrate tidb-tools into tidb

### PR DESCRIPTION
**Pull request by hawkingrei**
### What problem does this PR solve?

Issue Number: close #32999

Problem Summary:

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

**Review by disksing — APPROVED** [review]

**Comment by disksing** [comment]
TLDR:)

**Review by BornChanger — APPROVED** [review]

**Review by xhebox — APPROVED** [review]
